rust-src/wallet_library/src/credential.rs

```diff
@@ -82,11 +83,42 @@ pub fn create_unsigned_credential_with_seed_v1_aux(
         wallet.get_blinding_randomness(identity_provider_index, identity_index)?;
     let encoded_blinding_randomness = base16_encode_string(&blinding_randomness);
 
+    let identity_attributes_iter = input.common.id_object.get_attribute_list().alist.iter();
+    let revealed_attributes = &input.common.revealed_attributes;
+
+    let mut attribute_randomness = BTreeMap::new();
+    for (tag, _kind) in identity_attributes_iter {
+        if !revealed_attributes.contains(tag) {
+            let randomness: PedersenRandomness<ArCurve> = wallet
+                .get_attribute_commitment_randomness(
+                    identity_provider_index,
+                    identity_index,
+                    input.common.cred_number.into(),
+                    *tag,
+                )?;
+            attribute_randomness.insert(*tag, randomness);
+        }
+    }
+
+    let key = wallet.get_account_public_key(
+        identity_provider_index,
+        identity_index,
+        input.common.cred_number.into(),
+    )?;
+    let mut key_map: BTreeMap<KeyIndex, VerifyKey> = BTreeMap::new();
+    key_map.insert(KeyIndex(0), VerifyKey::Ed25519VerifyKey(key));
+    let credential_public_keys: CredentialPublicKeys = CredentialPublicKeys {
+        threshold: NonZeroThresholdU8::ONE,
+        keys:      key_map,
```

Comment:
Maybe it would be useful to explicitly state this in the documentation, the fact that this hardcodes the choice of threshold and number of keys.